src/main/java/com/google/devtools/build/lib/skyframe/ActionMetadataHandler.java

```diff
@@ -483,6 +485,18 @@ public void injectRemoteFile(Artifact output, byte[] digest, long size, int loca
     store.injectRemoteFile(output, digest, size, locationIndex);
   }
 
+  @Override
+  public void injectRemoteDirectory(SpecialArtifact output,
```

Comment:
Naming consistency - don't call this "output"?

Reply:
done